docs/self-hosted/config-options.md

```diff
@@ -255,6 +255,7 @@ into unexpected behaviors.
 | `DB_PASSWORD`              | Database user's password. **Required** when using `pg`, `mysql`, `oracledb`, or `mssql`.                                                           | --                            |
 | `DB_FILENAME`              | Where to read/write the SQLite database. **Required** when using `sqlite3`.                                                                        | --                            |
 | `DB_CONNECTION_STRING`     | When using `pg`, you can submit a connection string instead of individual properties. Using this will ignore any of the other connection settings. | --                            |
+| `DB_SSL__CA`               | Self signed cert (could be necessary for some installation like "DigitalOcean" or "Yandex" cloud)                                                       | --                            |
```

Comment:
Certificate isn't necessarily self-signed. I'd also rather omit the shoutout to specific hosting vendors, as requirements and configurations change over time out of our control